ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -222,6 +241,9 @@ A single compaction job could be performed across multiple compaction cycles (if
     logger.trace("resumeCompaction() started for {}", storeId);
     runningLatch = new CountDownLatch(1);
     compactionInProgress.set(true);
+    selectedContainers.clear();
```

Comment:
how about move this line into `getAllFilteredContainers `?

Reply:
Updated.